cpp-client/include/nvidia/aiaa/exception.h

```diff
@@ -51,7 +51,8 @@ namespace aiaa {
  nvidia.aiaa.error.103 | Failed to process ITK Operations.
  nvidia.aiaa.error.104 | Invalid Arguments.
  nvidia.aiaa.error.105 | AIAA Session Timeout.
- nvidia.aiaa.error.106 | System/Unknown Error.
+ nvidia.aiaa.error.106 | AIAA Response Error.
```

Comment:
Do we want to also call this AIAA Server Error to be consistent with py-client?

Reply:
these are more related to cpp errors and exceptions..  individual client lib-wise rather than server side..